desktop/src/shared-ui/filter-bar/filter-bar.component.html

```
@@ -0,0 +1,20 @@
+<div class="filter-bar" role="tablist">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)